src/initial_conditions.jl

```diff
@@ -0,0 +1,37 @@
+
+export WilliamsonInicond1
+
+"""
+$(TYPEDSIGNATURES)
+Initial conditions for the first test case of Williamson et al. 1992.
+α defaults to 0.0 and should be set to the transport direction being tested.
+$(TYPEDFIELDS)"""
+@kwdef struct WilliamsonInicond1 <: SpeedyWeather.AbstractInitialConditions
+   h0::Float64 = 1000 # m
+   θc::Float64 = 0.0 # degrees
+   λc::Float64 = 270.0 # degrees
+   α::Float64 # degrees
```

Comment:
if the default is 0 here, write it in? Then you don't have to write it into the doc string! Also if you do

```julia
"Some string to describe a variable called var, unit [kg]"
var::Float64 = 123.45
```

then the DocStringExtensions.jl we use will automatically pick this up and write it into `$(TYPEDFIELDS)` above! You can always check this with `?WilliamsonInicond1` or on an instance of it e.g. `w = WilliamsonInicond1; ?w`

Reply:
also you can write `= 270.0` or `= 270` or `= π` whatever is on the right will be converted to the type on the left, so there's an implicit `convert(Float64, ...)` call!